transformer.py

```diff
@@ -52,6 +57,98 @@
 object_store = None
 
 
+def initialize_logging(request=None):
+    """
+    Get a logger and initialize it so that it outputs the correct format
+
+    :param request: Request id to insert into log messages
+    :return: logger with correct formatting that outputs to console
+    """
+
+    log = logging.getLogger()
+    if 'INSTANCE_NAME' in os.environ:
+        instance = os.environ['INSTANCE_NAME']
+    else:
+        instance = 'Unknown'
```

Comment:
Prefer `.get` for `os.environ`, dictionaries, and other mappings.
```suggestion
    instance = os.environ.get('INSTANCE_NAME', 'Unknown')
```